components/molecules/intro.tsx

```diff
@@ -0,0 +1,11 @@
+export default function Intro ({ surtitle, title, text }) {
```

Comment:
What is the difference between intro and intro-home? I mean can't we figure out how to use only one for both use cases?